bundles/org.palladiosimulator.textual.tpcm.ui/src/org/palladiosimulator/textual/tpcm/ui/labeling/TPCMLabelProvider.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * generated by Xtext 2.21.0
+ */
+package org.palladiosimulator.textual.tpcm.ui.labeling;
+
+import com.google.inject.Inject;
+import org.eclipse.emf.edit.ui.provider.AdapterFactoryLabelProvider;
+import org.eclipse.xtext.ui.label.DefaultEObjectLabelProvider;
+
+/**
+ * Provides labels for EObjects.
+ * 
+ * See https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
+ */
+public class TPCMLabelProvider extends DefaultEObjectLabelProvider {
+
+	@Inject
+	public TPCMLabelProvider(AdapterFactoryLabelProvider delegate) {
```

Comment:
Do we really need this constructor? Also consider cleaning the generated example comment code

Reply:
The class is generated by Xtext and the constructor is needed for dependency injection. Otherwise, we would need to remove the class entirely and override the generated binding in the guice module. I would opt against it.

I shall, however, clean the generated commented example code.